### ISSUE TITLE
Riot video and voice chat does not work on Blackberry 10 phones. Fix this and attract a wide user base of Blackberry users.

### ISSUE THREAD

**Issue by Waxwood** [vector-im/riot-android#1048]
Hi, I've been using RIOT for 2 months after reading various security websites praise for Riot. I installed the Android version of Riot on my Blackberry 10 phone. The BB10 operating system can run Android apps with varying degrees of success. Riot installed easily using Fdroid. Overall, it runs well and sends text messages flawlessly, however the voice and video call features do not work. 

In order to popularise Riot, I'd like to suggest that Riot developers either A) make a port of the Android app for Blackberry 10 so it can be included in Blackberry World, or B) code the Android app so that it offers full functionality on Blackberry10 phones.

Blackberry 10 users will be thrilled I believe to have access to a secure open source app like Riot. WhatsApp is abandoning BB10 in the summer of 2017. Most of the other major chat apps have also stopped supporting BB10. However, there are still many hundreds of thousands of BB10 phones that will be in use for the foreseeable future. If Riot ran properly on BB10 phones, it seems very likely that many users will switch to Riot when WhatsApp shuts off support for BB10.

Blackberry users are very security conscious and will likely be keen on adopting Riot. Many BB10 users would like to use SIGNAL, but it will not run on Blackberry 10 because it requires GooglePlayServices.

In short, ensuring that Riot runs on Blackberry 10 phones would help convert many thousands of users to Riot because it would be the only secure messenger app available for Blackberry. (Blackberry's own messenger BBM is essentially irrelevant now because they refused to make cross-platform desktop apps and took too long to develop versions for iOS and Android)
